integration_tests/integration/amend-referral.cy.js

```diff
@@ -292,6 +292,10 @@ context('Amend a referral', () => {
         )
         cy.contains('What are the desired outcomes for Accommodation?')
 
+        cy.get('[data-cy=desired-outcomes]').within(() => {
```

Comment:
Do we have display this banner only for desired outcomes and not for any other amends?

Reply:
its desired outcomes only, that's the only amend that had this requirement (because the view change detail screen needs to split this into 'added' and 'removed' sections, so it wouldn't work if we allowed no changes to the list of outcomes)

Reply:
Ok sure @NeilLuptonMoJ